introduction.md

```diff
@@ -12,35 +12,58 @@ position: 0
+|:--|:--
+|ASP.NET Core|Web
+|ASP.NET MVC|Web
+|ASP.NET WebForms|Web
+|Angular|Web
+|React|Web
+|Blazor|Web
+|WinForms|Desktop
+|WPF|Desktop
+|WinUI|Desktop 
 
-Telerik Reporting can retrieve and process data from relational, multi-dimensional, ORM, or custom data-layer-based data sources. The end users can view the ready reports in PDF, Microsoft Office Word, Excel, and PowerPoint document formats, or use a dedicated viewer in a Web or .NET desktop application. 
 
 ## Key Components 
 
-The main components of Telerik Reporting are its designers and viewers. 
+The lifecycle of a Telerik report is divided in three main steps&mdash;designing the report, populating the report with data and rendering it in a viewer, and exporting the report to a document file.
+
+The Telerik Report Designers and Viewers are the main components of Telerik Reporting and accompany the users in their journey along the lifecycle of the report. 
 
 ### Report Designers 
 
-The Reporting designers are responsible for the designing process. They are tools which help you create the report you require, connect it to a data source to load the information you need, style its header, footer, and general appearance, make reports collections if necessary, and export its content to PDF, for example. Which designer you will choose depends on the specific functionality you want to use in the process.
+The Report Designers are responsible for the designing process. 
+
+* Report Designers are tools which help you create the report you require, connect it to a data source to load the information you need, style its header, footer, and general appearance, make reports collections if necessary, and export its content to PDF, for example. 
 
-The Telerik Reporting designer tools are: 
+* Which Report Designer you will choose depends on the specific functionality you want to use in the process.
 
-|Telerik Reporting Designer|Specifics
+|Report Designer|Specifics
 |:---|:---
 |Standalone desktop designer|<ul><li>A single `.exe` file representing a Windows tool powered by the Telerik Reporting engine.</li><li>Uses the latest UI standards to facilitate the process of creating reports, locally analyzing report data, exporting and sharing.</li></ul>
-|Web report designer|<ul><li>An HTML5-, JavaScript-, or CSS3 jQuery-based widget allowing you to integrate a report designer into your web applications.</li><li>Styling is based on the Kendo UI Sass Default theme.</li><li>Previews reports in the HTML5 Report Viewer.</li></ul>
+|Web report designer|<ul><li>A JavaScript-, CSS3 jQuery-based widget allowing you to integrate a report designer into your web applications.</li><li>Styling is based on the Kendo UI Sass Default theme.</li><li>Previews reports in the HTML5 Report Viewer.</li></ul>
 |Designer for Visual Studio|<ul><li>Dedicated to editing CLR or type report definitions (CS or VB files) in the Visual Studio environment.</li><li>Available under the .NET framework only.</li></ul>
```

Comment:
'.NET framework' may be changed to '.NET Framework' as now we have the old '.NET Framework', the '.NET Core' and the new only '.NET' like .NET 5/6/7. The capital letter 'F' in '.NET Framework' will say to the user that this is supported only for the old .NET Framework, which is what we want.